onnx/external_data_helper.py

```diff
@@ -222,11 +228,12 @@ def _recursive_attribute_processor(
 
 
 def _get_initializer_tensors_from_graph(
-    onnx_model_proto_graph: GraphProto,
+    graph_or_function: GraphProto | FunctionProto,
```

Comment:
```suggestion
    graph_or_function: GraphProto | FunctionProto, /
```

to avoid calls by keyword